src/pfs/pfs.proto

```diff
@@ -93,6 +93,7 @@ enum OriginKind {
   USER = 0;
   AUTO = 1;
   FSCK = 2;
+  ALIAS = 3;
```

Comment:
Added a separate commit type for alias commits so that we can avoid creating jobs on alias commits.